hevc-webrtc-04.nr

```diff
@@ -149,7 +149,9 @@ to be sent both in-band and out-of-band.  WebRTC implementations
 MUST signal this information in-band.  This means that WebRTC
 implementations MUST NOT include these parameters in the SDP they
 generate, and SHOULD silently ignore these parameters if they
-are received..
+are received. Any IDR sent MUST always be preceded by the relevant
+parameter sets sent in a packet (not necessarily a separate packet)
+with the same timestamp as the IDR.
```

Comment:
If we feel like this formulation is good enough for normative text then I have one last nit: replace "timestamp" with "RTP timestamp".

Other than that, LGTM :)